cdap-ui/app/cdap/services/datasource/index.js

```diff
@@ -116,18 +146,15 @@ export default class Datasource {
       type: 'REQUEST',
     };
 
-    Socket.send({
-      action: 'request',
-      resource: generatedResource,
-    });
+    this.socketSend('request', generatedResource);
 
     return subject;
   }
 
   poll(resource = {}) {
     let generatedResource = {
       id: uuidV4(),
-      interval: resource.interval || 10000,
+      interval: resource.interval || 1000,
```

Comment:
Was this intended change?

Reply:
no, it was to ease testing, will be reverted